src/Leaderboard.ts

```diff
@@ -177,4 +191,8 @@ export default class Leaderboard {
     // update all leaderbaord messages
     for (const msg of this._messages) msg.edit(newMsg).catch(console.error);
   }
+
+  private getDay(now: Date, day: number): Date {
```

Comment:
The function name `getDay` might lead to confusion since this function returns a date. Maybe `getDateOfChallengeBegin`? (If I understood the code correctly then this is what this function does)

Reply:
yeah... i'll do that